publisher/mail/_mailer.py

```
@@ -76,3 +122,170 @@ def send_template(sender, recipient, template, template_data,
 
     session.sendmail(sender['name'], recipient, message)
     session.quit()
+
+
+def load_data_file(source_file):
+        if os.path.exists(source_file):
+            return cfg2dict(source_file)
+        else:
+             print('file at {} not found'.format(os.path.abspath(s)))
```

If the data file doesn't exist, would it be better to raise an exception since the dependent functions cannot do what they need to do otherwise?

---

yes, I'll make that happen.